packages/sdk/src/evm/contracts/smart-contract.ts

```diff
@@ -273,6 +283,12 @@ export class SmartContract<TContract extends BaseContract = BaseContract>
     return undefined;
   }
 
+  private detectAppURI() {
+    if (detectContractFeature<AppURI>(this.contractWrapper, "Ownable")) {
```

Comment:
This should be detecting the `AppURI` feature - not `Ownable`

Reply:
fixed in latest